lib/builders.js

```diff
@@ -99,6 +130,128 @@ class JSONBuilder {
       console.log(chalk.green('Wrote JSON licenses to licenses.json'));
     });
   }
+
+  _notifyMissing() {
+    let empties = [],
+      missingRepos = [];
+    _.forEach(this.output, (pkg, pkgName) => {
+      if (!pkg[_.keys(_.omit(pkg, 'repository'))[0]]) {
+        empties.push(pkgName);
+      } else if (!pkg.repository) {
+        missingRepos.push(pkgName);
+      }
+    });
+
+    if (this.shouldNotifyLinks && !_.isEmpty(missingRepos)) {
+      console.log(
+        chalk.yellow(
+          `The following packages did not have a verifiable online LICENSE link: ${missingRepos
+            .toString()
+            .replace(',', '\n')}`
+        )
+      );
+    }
+    if (!_.isEmpty(empties)) {
+      console.log(
+        chalk.red(
+          `The following packages did not have a local LICENSE file OR a LICENSE file/text in cache: ${empties
+            .toString()
+            .replace(',', '\n')}`
+        )
+      );
+    }
+  }
+}
+
+class HTMLBuilder {
+  constructor() {
```

Comment:
Since this is pretty simple I think doing it string based is fine, if it gets any more complex might want to consider using something like `mustache`